witness/src/Dscp/Witness/Web/Types.hs

```diff
@@ -107,6 +108,9 @@ data HashIs
 deriving instance (Eq (Id a), Eq a) => Eq (PaginatedList d a)
 deriving instance (Show (Id a), Show a) => Show (PaginatedList d a)
 
+instance Buildable ByteString where
+    build = (build :: Text -> Builder) . decodeUtf8 . Base64.encode
```

Comment:
Maybe we do not always want to see them in base64? I think there is a reason why `instance Buildable ByteString` is not defined in the original library. 
Your data extracted from PDF seems to have a quite specific format, would be nice to see a newtype carried around.

Reply:
Actually, that's the PDF itself. I'm thinking about `build _ = "<binary-data>"`, if the output of this ever goes to any log.

Reply:
Ah, right.
We really do log it since any call of an endpoint is entirely logged.
I can imagine which trash our logs will turn into if we print the entire PDF :smile: